### PR TITLE
Fix/editors

### PR DESCRIPTION
**Pull request by VladMartyshevskyi**
1. Improved CSS styles for disabled buttons.
2. Fixed null versions in the version history.

### CODE REVIEW

File: core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java [hunk 1]
```diff
@@ -807,4 +811,88 @@ public static String getLinkInDocumentsApplication(String nodePath) throws Excep
     DocumentService documentService = WCMCoreUtils.getService(DocumentService.class);
     return documentService.getLinkInDocumentsApp(nodePath);
   }
+  
+  /**
+   * Invoke user session.
```
Comment (pnedonosko) [inline core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java:816]: We need real Javadoc describing what is it and what it does.

File: core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java [hunk 2]
```diff
@@ -807,4 +811,88 @@ public static String getLinkInDocumentsApplication(String nodePath) throws Excep
...
+   *
+   * @param userId the user id
+   * @param uuid the uuid
+   * @param workspace the workspace
+   * @param handler the handler
+   * @throws RepositoryException the repository exception
+   */
+  @SuppressWarnings("deprecation")
+  public static void invokeUserSession(String userId, String uuid, String workspace, RepositoryConsumer<Node> handler) throws RepositoryException {
```
Comment (pnedonosko) [inline core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java:825]: Work with JCR Session not Node, just user and function generalized by Session. This way this method will be generally used. Currently it is only for node by UUID cases.

File: core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java [hunk 3]
```diff
@@ -807,4 +811,88 @@ public static String getLinkInDocumentsApplication(String nodePath) throws Excep
...
+  
+  /**
+   * Node by UUID.
+   *
+   * @param uuid the uuid
+   * @param workspace the workspace
+   * @return the node
+   * @throws RepositoryException the repository exception
+   */
+  public static Node nodeByUUID(String uuid, String workspace) throws RepositoryException {
```
Comment (pnedonosko) [inline core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java:861]: Don't need this method here.

File: core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java [hunk 4]
```diff
@@ -807,4 +811,88 @@ public static String getLinkInDocumentsApplication(String nodePath) throws Excep
...
+  }
+  
+  /**
+   * Find or create user identity.
+   *
+   * @param userId the user id
+   * @return the identity can be null if not found and cannot be created via
+   *         current authenticator
+   */
+  private static Identity userIdentity(String userId) {
```
Comment (pnedonosko) [inline core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java:879]: Avoid this method - do what you need where you need.

File: core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java (hunk 1 above)
Comment (VladMartyshevskyi) [reply core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java]: Fixed

File: core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java (hunk 3 above)
Comment (VladMartyshevskyi) [reply core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java]: Removed this method.

File: core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java (hunk 4 above)
Comment (VladMartyshevskyi) [reply core/services/src/main/java/org/exoplatform/services/wcm/utils/WCMCoreUtils.java]: Removed this method.

**Pull request merged**